### PR TITLE
Add Quadrix as client

### PR DESCRIPTION
**Pull request by alariej**
Quadrix is a minimal, simple, multi-platform Matrix client programmed with ReactXP.

Repo: https://github.com/alariej/quadrix

Thanks!

Preview: https://pr1348--matrix-org-previews.netlify.app

### CODE REVIEW

**Comment by alariej** [comment]
Hi, is there anything I need to do for this PR to be considered? Thanks and sorry for the ping.

**Comment by alariej** [comment]
Let's close this sucker, looks like it won't be merged anyway..

**Comment by alariej** [comment]
Hi @MadLittleMods, I fixed the spelling which had failed in the previous check. Anything else I need to do? Any idea when this could be merged? Thanks.

File: gatsby/content/projects/clients/quadrix.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    SSO Support: no
+---  
+The current focus is on creating a simple and stable chat application
+which runs on all platforms. The Quadrix apps are available on Linux
+(snap and flatpak packages for amd64, arm64, armhf), Windows, Mac,
+Android and iOS. It has been tested on a PinePhone running Mobian/Phosh.
+There is no VoIP support yet, but Quadrix does offer in-app
+videoconferencing with Jitsi Meet. Please visit the Github repo for
+more information. Feel free to open an issue on Github, or leave a
+comment at #quadrix:matrix.org
```
Comment (MadLittleMods) [inline gatsby/content/projects/clients/quadrix.mdx:66]:
```suggestion
videoconferencing with Jitsi Meet. Please visit the GitHub repo for
more information. Feel free to open an issue on GitHub, or leave a
comment at [#quadrix:matrix.org](https://matrix.to/#/#quadrix:matrix.org)
```

File: gatsby/content/projects/clients/quadrix.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    Reactions: no
+    Message editing: no
+    Room upgrades: no
+    Localisations: 3
+    SSO Support: no
+---  
+The current focus is on creating a simple and stable chat application
+which runs on all platforms. The Quadrix apps are available on Linux
+(snap and flatpak packages for amd64, arm64, armhf), Windows, Mac,
+Android and iOS. It has been tested on a PinePhone running Mobian/Phosh.
```
Comment (MadLittleMods) [inline gatsby/content/projects/clients/quadrix.mdx:62]: Cool to see PinePhone getting some more support! 🙂

File: gatsby/content/projects/clients/quadrix.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+maturity: Released
+platforms:
+    - Linux
+    - Mac
+    - Windows
+    - Web
+    - Android
+    - iOS
+    - Mobian
+featured: true
```
Comment (MadLittleMods) [inline gatsby/content/projects/clients/quadrix.mdx:25]: Tested locally and the project page looks good:

http://localhost:8000/docs/projects/client/quadrix

Comment (MadLittleMods) [inline gatsby/content/projects/clients/quadrix.mdx:25]: Tested locally and the `quadrix.chat` project sorts really high compared to the existing Matrix clients in all categories. I know we could mess with `sort_order` but ideally [we don't want to mess with that and set it for everything](https://github.com/matrix-org/matrix.org/pull/1015).

File: gatsby/content/projects/clients/quadrix.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
+---
```
Comment (MadLittleMods) [inline gatsby/content/projects/clients/quadrix.mdx:1]: Fix formatting:

```mdx
---
layout: projectimage
title: quadrix.chat
slug: quadrix
categories:
  - client
description: Minimal Matrix client available in all main app stores
author: Jean-François Alarie
home: https://quadrix.chat
language: Typescript
license: GPL-3.0-or-later
repo: https://github.com/alariej/quadrix
room: "#quadrix:matrix.org"
thumbnail: /docs/projects/images/quadrix-screenshot.png
screenshot: /docs/projects/images/quadrix-screenshot.png
maturity: Released
platforms:
  - Linux
  - Mac
  - Windows
  - Web
  - Android
  - iOS
  - Mobian
featured: true
features:
  Room directory: yes
  Room tag showing: no
  Room tag editing: no
  Search joined rooms: no
  Room user list: yes
  Display Room Description: yes
  Edit Room Description: no
  Highlights: no
  Pushrules: no
  Send read markers: yes
  Display read markers: yes
  Sending Invites: yes
  Accepting Invites: yes
  Typing Notification: no
  E2E: no
  Replies: yes
  Attachment uploading: yes
  Attachment downloading: yes
  Send stickers: no
  Send formatted messages (markdown): no
  Rich Text Editor for formatted messages: no
  Display formatted messages: no
  Redacting: yes
  Multiple Matrix Accounts: no
  New user registration: yes
  voip: no
  Reactions: no
  Message editing: no
  Room upgrades: no
  Localisations: 3
  SSO Support: no
---

The current focus is on creating a simple and stable chat application
which runs on all platforms. The Quadrix apps are available on Linux
(snap and flatpak packages for amd64, arm64, armhf), Windows, Mac,
Android and iOS. It has been tested on a PinePhone running Mobian/Phosh.
There is no VoIP support yet, but Quadrix does offer in-app
videoconferencing with Jitsi Meet. Please visit the GitHub repo for
more information. Feel free to open an issue on GitHub, or leave a
comment at [#quadrix:matrix.org](https://matrix.to/#/#quadrix:matrix.org)
````

File: gatsby/content/projects/clients/quadrix.mdx (hunk 3 above)
Comment (MadLittleMods) [reply gatsby/content/projects/clients/quadrix.mdx:25]: This point may not matter. It does sort last (more as expected) after [fixing the formatting](https://github.com/matrix-org/matrix.org/pull/1348/files#r940673139).

File: gatsby/content/projects/clients/quadrix.mdx (hunk 2 above)
Comment (alariej) [reply gatsby/content/projects/clients/quadrix.mdx:62]: The flatpak runs surprisingly well on PinePhone. Main problem is the phone's camera, which refuses to start when launching a Jitsi call :-(

**Comment by MadLittleMods** [comment]
Thanks for adding your project @alariej 🐊

**Pull request merged**